2010/11/10 21:29:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/screen.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/screen.css
2010/11/10 21:29:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/print.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/print.css
2010/11/10 21:29:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/main.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/main.css
2010/11/10 21:29:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/form.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/form.css
2010/11/10 21:29:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/screen.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/screen.css
2010/11/10 21:29:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/print.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/print.css
2010/11/10 21:29:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/main.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/main.css
2010/11/10 21:29:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/form.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/form.css
2010/11/10 21:29:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/screen.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/screen.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/screen.css
2010/11/10 21:29:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/print.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/print.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/print.css
2010/11/10 21:29:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/main.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/main.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/main.css
2010/11/10 21:29:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "css/form.css".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('css/form.css')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/css/form.css
2010/11/10 21:34:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection falhou ao abrir uma conecção com o banco de dados: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbConnection.php:269
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\CDbConnection.php(242): CDbConnection->open()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\CDbConnection.php(221): CDbConnection->setActive(true)
#2 C:\localhost\yii-1.1.4.r2429\framework\base\CModule.php(363): CDbConnection->init()
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModule.php(86): CModule->getComponent('db')
#4 C:\localhost\yii-1.1.4.r2429\framework\gii\generators\model\ModelCode.php(51): CModule->__get('db')
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\localhost\yii-1.1.4.r2429\framework\gii\CCodeGenerator.php(149): CFormModel->__construct()
#7 C:\localhost\yii-1.1.4.r2429\framework\gii\CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('')
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('gii/model')
#14 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#15 C:\localhost\gertcc\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/gii/model
2010/11/10 21:46:38 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.nome" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('nome')
#1 C:\localhost\gertcc\protected\components\UserIdentity.php(34): CActiveRecord->__get('nome')
#2 C:\localhost\gertcc\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\localhost\yii-1.1.4.r2429\framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(LoginForm), NULL)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(591): CModel->validate(NULL)
#7 C:\localhost\gertcc\protected\controllers\SiteController.php(79): CActiveForm::validate(Object(LoginForm))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('login')
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/login')
#14 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#15 C:\localhost\gertcc\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/site/login
2010/11/10 21:46:40 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.nome" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('nome')
#1 C:\localhost\gertcc\protected\components\UserIdentity.php(34): CActiveRecord->__get('nome')
#2 C:\localhost\gertcc\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\localhost\yii-1.1.4.r2429\framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(LoginForm), NULL)
#6 C:\localhost\gertcc\protected\controllers\SiteController.php(88): CModel->validate()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('login')
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/login')
#13 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#14 C:\localhost\gertcc\index.php(13): CApplication->run()
#15 {main} REQUEST_URI=/site/login
2010/11/10 21:46:40 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.nome" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('nome')
#1 C:\localhost\gertcc\protected\components\UserIdentity.php(34): CActiveRecord->__get('nome')
#2 C:\localhost\gertcc\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\localhost\yii-1.1.4.r2429\framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(LoginForm), NULL)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(591): CModel->validate(NULL)
#7 C:\localhost\gertcc\protected\controllers\SiteController.php(79): CActiveForm::validate(Object(LoginForm))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('login')
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/login')
#14 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#15 C:\localhost\gertcc\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/site/login
2010/11/10 21:51:26 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.nome" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('nome')
#1 C:\localhost\gertcc\protected\components\UserIdentity.php(34): CActiveRecord->__get('nome')
#2 C:\localhost\gertcc\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\localhost\yii-1.1.4.r2429\framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(LoginForm), NULL)
#6 C:\localhost\gertcc\protected\controllers\SiteController.php(88): CModel->validate()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('login')
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/login')
#13 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#14 C:\localhost\gertcc\index.php(13): CApplication->run()
#15 {main} REQUEST_URI=/site/login
2010/11/10 21:51:27 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.nome" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('nome')
#1 C:\localhost\gertcc\protected\components\UserIdentity.php(34): CActiveRecord->__get('nome')
#2 C:\localhost\gertcc\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\localhost\yii-1.1.4.r2429\framework\validators\CInlineValidator.php(39): LoginForm->authenticate('password', Array)
#4 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(LoginForm), NULL)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(591): CModel->validate(NULL)
#7 C:\localhost\gertcc\protected\controllers\SiteController.php(79): CActiveForm::validate(Object(LoginForm))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionLogin()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('login')
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/login')
#14 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#15 C:\localhost\gertcc\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/site/login
2010/11/10 22:00:52 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:01:17 [error] [exception.CException] exception 'CException' with message 'A propriedade "AnexoController.portletTitle" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:173
Stack trace:
#0 C:\localhost\gertcc\protected\views\anexo\create.php(7): CComponent->__set('portletTitle', 'Anexos')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('create', Array, true)
#5 C:\localhost\gertcc\protected\controllers\AnexoController.php(78): CController->render('create', Array)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#17 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#18 C:\localhost\gertcc\index.php(13): CApplication->run()
#19 {main} REQUEST_URI=/anexo/create
2010/11/10 22:01:58 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:02:07 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:08:17 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:19:19 [error] [exception.CException] exception 'CException' with message 'A propriedade "AnexoController.portletTitle" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:173
Stack trace:
#0 C:\localhost\gertcc\protected\views\anexo\create.php(7): CComponent->__set('portletTitle', 'Anexos')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('create', Array, true)
#5 C:\localhost\gertcc\protected\controllers\AnexoController.php(74): CController->render('create', Array)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#17 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#18 C:\localhost\gertcc\index.php(13): CApplication->run()
#19 {main} REQUEST_URI=/anexo/create
2010/11/10 22:21:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "images/criar.png".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('images/criar.pn...')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/images/criar.png
2010/11/10 22:23:18 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:23:34 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:23:35 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:41:28 [warning] [application] Failed to set unsafe attribute "caminho".
in C:\localhost\gertcc\protected\controllers\AnexoController.php (67)
in C:\localhost\gertcc\index.php (13)
2010/11/10 22:41:28 [warning] [application] Failed to set unsafe attribute "nome".
in C:\localhost\gertcc\protected\controllers\AnexoController.php (67)
in C:\localhost\gertcc\index.php (13)
2010/11/10 22:41:28 [warning] [application] Failed to set unsafe attribute "tipo".
in C:\localhost\gertcc\protected\controllers\AnexoController.php (67)
in C:\localhost\gertcc\index.php (13)
2010/11/10 22:41:28 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.url" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('url')
#1 C:\localhost\yii-1.1.4.r2429\framework\validators\CStringValidator.php(66): CActiveRecord->__get('url')
#2 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CStringValidator->validateAttribute(Object(Anexo), 'url')
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(Anexo), NULL)
#4 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(758): CModel->validate(NULL)
#5 C:\localhost\gertcc\protected\controllers\AnexoController.php(68): CActiveRecord->save()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#17 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#18 C:\localhost\gertcc\index.php(13): CApplication->run()
#19 {main} REQUEST_URI=/anexo/create
2010/11/10 22:41:54 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Você não está autorizado a realizar essa operação.' in C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php:153
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\auth\CAccessControlFilter.php(115): CAccessControlFilter->accessDenied(Object(CWebUser), 'Voc?? n??o est?...')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(39): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#3 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#4 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/anexo/admin
2010/11/10 22:48:21 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.url" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('url')
#1 C:\localhost\gertcc\protected\models\Anexo.php(84): CActiveRecord->__get('url')
#2 C:\localhost\gertcc\protected\views\anexo\admin.php(42): Anexo->search()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('admin', Array, true)
#7 C:\localhost\gertcc\protected\controllers\AnexoController.php(139): CController->render('admin', Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionAdmin()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#14 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('admin')
#18 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/admin')
#19 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#20 C:\localhost\gertcc\index.php(13): CApplication->run()
#21 {main} REQUEST_URI=/anexo/admin
2010/11/10 23:02:11 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.id_usuario" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('id_usuario')
#1 C:\localhost\gertcc\protected\controllers\AnexoController.php(177): CActiveRecord->__get('id_usuario')
#2 C:\localhost\gertcc\protected\controllers\AnexoController.php(45): AnexoController->loadModelByUser('3')
#3 [internal function]: AnexoController->actionView('3')
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(47): ReflectionMethod->invokeArgs(Object(AnexoController), Array)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('view')
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/view')
#15 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#16 C:\localhost\gertcc\index.php(13): CApplication->run()
#17 {main} REQUEST_URI=/anexo/3
2010/11/10 23:10:37 [error] [system.db.CDbCommand] Error in executing SQL: DELETE FROM `Anexo` WHERE `Anexo`.`id_anexo`=4
in C:\localhost\gertcc\protected\controllers\AnexoController.php (124)
in C:\localhost\gertcc\index.php (13)
2010/11/10 23:10:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`gertcc`.`pessoa_anexo`, CONSTRAINT `fk_pessoa_has_anexo_anexo1` FOREIGN KEY (`id_anexo`) REFERENCES `anexo` (`id_anexo`) ON DELETE NO ACTION ON UPDATE NO ACTION)' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1644): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1113): CActiveRecord->deleteByPk('4')
#2 C:\localhost\gertcc\protected\controllers\AnexoController.php(124): CActiveRecord->delete()
#3 [internal function]: AnexoController->actionDelete('4')
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(47): ReflectionMethod->invokeArgs(Object(AnexoController), Array)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('delete')
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/delete')
#15 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#16 C:\localhost\gertcc\index.php(13): CApplication->run()
#17 {main} REQUEST_URI=/anexo/delete/4?ajax=anexo-grid
2010/11/16 21:51:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "favicon.ico".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('favicon.ico')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/favicon.ico
2010/11/16 21:53:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "usuario".' in C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php:328
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('usuario')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#2 C:\localhost\gertcc\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/usuario
2010/11/16 22:14:41 [error] [exception.CException] exception 'CException' with message 'Usuario não possui um método chamado "signup".' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:266
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(195): CComponent->__call('signup', Array)
#1 C:\localhost\gertcc\protected\controllers\SiteController.php(100): CActiveRecord->__call('signup', Array)
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(100): Usuario->signup()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionSignup()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('signup')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/signup')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/signup
2010/11/16 22:32:50 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.confirma_senha_repeat" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('confirma_senha_...')
#1 C:\localhost\yii-1.1.4.r2429\framework\validators\CCompareValidator.php(84): CActiveRecord->__get('confirma_senha_...')
#2 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CCompareValidator->validateAttribute(Object(Usuario), 'confirma_senha')
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(Usuario), NULL)
#4 C:\localhost\gertcc\protected\controllers\SiteController.php(100): CModel->validate()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#11 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#12 C:\localhost\gertcc\index.php(13): CApplication->run()
#13 {main} REQUEST_URI=/site/registro
2010/11/16 22:33:07 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.confirma_senha_repeat" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('confirma_senha_...')
#1 C:\localhost\yii-1.1.4.r2429\framework\validators\CCompareValidator.php(84): CActiveRecord->__get('confirma_senha_...')
#2 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CCompareValidator->validateAttribute(Object(Usuario), 'confirma_senha')
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(Usuario), NULL)
#4 C:\localhost\gertcc\protected\controllers\SiteController.php(100): CModel->validate()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#11 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#12 C:\localhost\gertcc\index.php(13): CApplication->run()
#13 {main} REQUEST_URI=/site/registro
2010/11/16 22:33:39 [error] [exception.CException] exception 'CException' with message 'A propriedade "Usuario.senha_repeat" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('senha_repeat')
#1 C:\localhost\yii-1.1.4.r2429\framework\validators\CCompareValidator.php(84): CActiveRecord->__get('senha_repeat')
#2 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CCompareValidator->validateAttribute(Object(Usuario), 'senha')
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(Usuario), NULL)
#4 C:\localhost\gertcc\protected\controllers\SiteController.php(100): CModel->validate()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#11 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#12 C:\localhost\gertcc\index.php(13): CApplication->run()
#13 {main} REQUEST_URI=/site/registro
2010/11/16 22:42:45 [error] [php] Undefined index:  senha (C:\localhost\gertcc\protected\controllers\SiteController.php:101)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): SiteController->runAction()
#1 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): SiteController->runActionWithFilters()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): SiteController->run()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#4 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#5 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/site/registro
in C:\localhost\gertcc\protected\controllers\SiteController.php (101)
in C:\localhost\gertcc\index.php (13)
2010/11/16 22:43:36 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\controllers\SiteController.php (104)
in C:\localhost\gertcc\index.php (13)
2010/11/16 22:43:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1364 Field 'id_pessoa' doesn't have a default value' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1000): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(759): CActiveRecord->insert(NULL)
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(104): CActiveRecord->save()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/registro
2010/11/16 22:51:13 [error] [php] include(range.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\localhost\yii-1.1.4.r2429\framework\YiiBase.php:341)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(158): import()
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(259): createValidator()
#2 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(233): RegistroForm->createValidators()
#3 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(277): RegistroForm->getValidators()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(1161): RegistroForm->isAttributeRequired()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(446): activeLabelEx()
#6 C:\localhost\gertcc\protected\views\site\registro.php(13): CActiveForm->labelEx()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): SiteController->renderInternal()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): SiteController->renderFile()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): SiteController->renderPartial()
#11 C:\localhost\gertcc\protected\controllers\SiteController.php(100): SiteController->render()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): SiteController->runAction()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): SiteController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): SiteController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/site/registro
in C:\localhost\gertcc\protected\views\site\registro.php (13)
in C:\localhost\gertcc\protected\controllers\SiteController.php (100)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:00:02 [error] [exception.CException] exception 'CException' with message 'A propriedade "RegistroForm.confirma_senha" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:173
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(447): CComponent->__set('confirma_senha', '')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php(152): CModel->setAttributes(Array)
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(97): CComponent->__set('attributes', Array)
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/registro
2010/11/16 23:00:21 [error] [exception.CException] exception 'CException' with message 'A propriedade "RegistroForm.confirma_senha" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:173
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(447): CComponent->__set('confirma_senha', '')
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php(152): CModel->setAttributes(Array)
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(97): CComponent->__set('attributes', Array)
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/registro
2010/11/16 23:00:43 [error] [exception.CException] exception 'CException' with message 'A propriedade "RegistroForm.confirma_senha" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\validators\CStringValidator.php(66): CComponent->__get('confirma_senha')
#1 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CStringValidator->validateAttribute(Object(RegistroForm), 'confirma_senha')
#2 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CValidator->validate(Object(RegistroForm), NULL)
#3 C:\localhost\gertcc\protected\controllers\SiteController.php(98): CModel->validate()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#11 C:\localhost\gertcc\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/site/registro
2010/11/16 23:11:40 [error] [php] preg_match() [<a href='function.preg-match'>function.preg-match</a>]: No ending delimiter '^' found (C:\localhost\yii-1.1.4.r2429\framework\validators\CEmailValidator.php:85)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\validators\CValidator.php(184): CEmailValidator->validateAttribute()
#1 C:\localhost\yii-1.1.4.r2429\framework\base\CModel.php(150): CEmailValidator->validate()
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->validate()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): SiteController->runAction()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): SiteController->runActionWithFilters()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): SiteController->run()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/site/registro
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:12:39 [error] [exception.CException] exception 'CException' with message 'RegistroForm não possui um método chamado "save".' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:266
Stack trace:
#0 C:\localhost\gertcc\protected\controllers\SiteController.php(98): CComponent->__call('save', Array)
#1 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->save()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#9 C:\localhost\gertcc\index.php(13): CApplication->run()
#10 {main} REQUEST_URI=/site/registro
2010/11/16 23:26:50 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:26:50 [error] [exception.CException] exception 'CException' with message 'CDbConnection não possui um método chamado "rollBack".' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:266
Stack trace:
#0 C:\localhost\gertcc\protected\models\RegistroForm.php(73): CComponent->__call('rollBack', Array)
#1 C:\localhost\gertcc\protected\models\RegistroForm.php(73): CDbConnection->rollBack()
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/registro
2010/11/16 23:28:47 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:28:52 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:28:55 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:29:04 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:29:22 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:29:45 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:29:49 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/16 23:30:17 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:36:11 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:38:27 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (68)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:39:49 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (70)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:40:52 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (70)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:41:11 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (70)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:41:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1364 Field 'id_pessoa' doesn't have a default value' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1000): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(759): CActiveRecord->insert(NULL)
#2 C:\localhost\gertcc\protected\models\RegistroForm.php(70): CActiveRecord->save()
#3 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#11 C:\localhost\gertcc\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/site/registro
2010/11/17 07:42:26 [warning] [application] Failed to set unsafe attribute "id_pessoa".
in C:\localhost\gertcc\protected\models\RegistroForm.php (65)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:42:26 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (72)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:42:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1364 Field 'id_pessoa' doesn't have a default value' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1000): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(759): CActiveRecord->insert(NULL)
#2 C:\localhost\gertcc\protected\models\RegistroForm.php(72): CActiveRecord->save()
#3 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#11 C:\localhost\gertcc\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/site/registro
2010/11/17 07:43:04 [warning] [application] Failed to set unsafe attribute "id_pessoa".
in C:\localhost\gertcc\protected\models\RegistroForm.php (65)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:43:04 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (72)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:43:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1364 Field 'id_pessoa' doesn't have a default value' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1000): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(759): CActiveRecord->insert(NULL)
#2 C:\localhost\gertcc\protected\models\RegistroForm.php(72): CActiveRecord->save()
#3 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#11 C:\localhost\gertcc\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/site/registro
2010/11/17 07:46:11 [warning] [application] Failed to set unsafe attribute "id_pessoa".
in C:\localhost\gertcc\protected\models\RegistroForm.php (65)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:46:11 [error] [system.db.CDbCommand] Error in executing SQL: INSERT INTO `usuario` (`login`, `email`, `senha`) VALUES (:yp0, :yp1, :yp2)
in C:\localhost\gertcc\protected\models\RegistroForm.php (70)
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 07:46:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falhou ao executar o comando SQL: SQLSTATE[HY000]: General error: 1364 Field 'id_pessoa' doesn't have a default value' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbCommand.php:234
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(1000): CDbCommand->execute()
#1 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(759): CActiveRecord->insert(NULL)
#2 C:\localhost\gertcc\protected\models\RegistroForm.php(70): CActiveRecord->save()
#3 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#10 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#11 C:\localhost\gertcc\index.php(13): CApplication->run()
#12 {main} REQUEST_URI=/site/registro
2010/11/17 07:51:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbTransaction está inativo e não pode realziar as operações commit ou roll back.' in C:\localhost\yii-1.1.4.r2429\framework\db\CDbTransaction.php:82
Stack trace:
#0 C:\localhost\gertcc\protected\models\RegistroForm.php(79): CDbTransaction->rollback()
#1 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#9 C:\localhost\gertcc\index.php(13): CApplication->run()
#10 {main} REQUEST_URI=/site/registro
2010/11/17 07:53:48 [error] [exception.CException] exception 'CException' with message 'RegistroForm não possui um método chamado "redirect".' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:266
Stack trace:
#0 C:\localhost\gertcc\protected\models\RegistroForm.php(82): CComponent->__call('redirect', Array)
#1 C:\localhost\gertcc\protected\models\RegistroForm.php(82): RegistroForm->redirect('login')
#2 C:\localhost\gertcc\protected\controllers\SiteController.php(98): RegistroForm->registrar()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): SiteController->actionRegistro()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('registro')
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('site/registro')
#9 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#10 C:\localhost\gertcc\index.php(13): CApplication->run()
#11 {main} REQUEST_URI=/site/registro
2010/11/17 20:54:21 [error] [php] Indirect modification of overloaded property RegistroForm::$attributes has no effect (C:\localhost\gertcc\protected\controllers\SiteController.php:98)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(278): SiteController->runAction()
#1 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): SiteController->runActionWithFilters()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): SiteController->run()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#4 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#5 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/site/registro
in C:\localhost\gertcc\protected\controllers\SiteController.php (98)
in C:\localhost\gertcc\index.php (13)
2010/11/17 21:15:25 [error] [php] Undefined variable: maxfilesize (C:\localhost\gertcc\protected\views\anexo\_form.php:14)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#1 C:\localhost\gertcc\protected\views\anexo\create.php(15): AnexoController->renderPartial()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): AnexoController->renderInternal()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): AnexoController->renderPartial()
#6 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): AnexoController->render()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): AnexoController->runAction()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CAccessControlFilter->filter()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): AnexoController->filterAccessControl()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): AnexoController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): AnexoController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/anexo/create
in C:\localhost\gertcc\protected\views\anexo\_form.php (14)
in C:\localhost\gertcc\protected\views\anexo\create.php (15)
in C:\localhost\gertcc\protected\controllers\AnexoController.php (88)
2010/11/17 21:17:55 [error] [php] Undefined variable: maxfilesize (C:\localhost\gertcc\protected\views\anexo\_form.php:14)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#1 C:\localhost\gertcc\protected\views\anexo\create.php(15): AnexoController->renderPartial()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): AnexoController->renderInternal()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): AnexoController->renderPartial()
#6 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): AnexoController->render()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): AnexoController->runAction()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CAccessControlFilter->filter()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): AnexoController->filterAccessControl()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): AnexoController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): AnexoController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/anexo/create
in C:\localhost\gertcc\protected\views\anexo\_form.php (14)
in C:\localhost\gertcc\protected\views\anexo\create.php (15)
in C:\localhost\gertcc\protected\controllers\AnexoController.php (88)
2010/11/17 21:20:15 [error] [php] Undefined variable: maxfilesize (C:\localhost\gertcc\protected\views\anexo\_form.php:14)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#1 C:\localhost\gertcc\protected\views\anexo\create.php(15): AnexoController->renderPartial()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): AnexoController->renderInternal()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): AnexoController->renderPartial()
#6 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): AnexoController->render()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): AnexoController->runAction()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CAccessControlFilter->filter()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): AnexoController->filterAccessControl()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): AnexoController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): AnexoController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/anexo/create
in C:\localhost\gertcc\protected\views\anexo\_form.php (14)
in C:\localhost\gertcc\protected\views\anexo\create.php (15)
in C:\localhost\gertcc\protected\controllers\AnexoController.php (88)
2010/11/17 21:20:31 [error] [php] Undefined variable: maxfilesize (C:\localhost\gertcc\protected\views\anexo\_form.php:13)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#1 C:\localhost\gertcc\protected\views\anexo\create.php(15): AnexoController->renderPartial()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): AnexoController->renderInternal()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): AnexoController->renderPartial()
#6 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): AnexoController->render()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): AnexoController->runAction()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CAccessControlFilter->filter()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): AnexoController->filterAccessControl()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): AnexoController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): AnexoController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/anexo/create
in C:\localhost\gertcc\protected\views\anexo\_form.php (13)
in C:\localhost\gertcc\protected\views\anexo\create.php (15)
in C:\localhost\gertcc\protected\controllers\AnexoController.php (88)
2010/11/17 21:22:55 [error] [php] Undefined variable: maxfilesize (C:\localhost\gertcc\protected\views\anexo\_form.php:14)
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#1 C:\localhost\gertcc\protected\views\anexo\create.php(15): AnexoController->renderPartial()
#2 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require()
#3 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): AnexoController->renderInternal()
#4 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): AnexoController->renderFile()
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): AnexoController->renderPartial()
#6 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): AnexoController->render()
#7 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#8 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#9 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): AnexoController->runAction()
#10 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CAccessControlFilter->filter()
#12 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): AnexoController->filterAccessControl()
#13 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): AnexoController->runActionWithFilters()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): AnexoController->run()
#17 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController()
#18 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#19 C:\localhost\gertcc\index.php(13): CWebApplication->run()
REQUEST_URI=/anexo/create
in C:\localhost\gertcc\protected\views\anexo\_form.php (14)
in C:\localhost\gertcc\protected\views\anexo\create.php (15)
in C:\localhost\gertcc\protected\controllers\AnexoController.php (88)
2010/11/17 21:23:34 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.tipo_arquivo" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('tipo_arquivo')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(2006): CActiveRecord->__get('tipo_arquivo')
#2 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(1379): CHtml::resolveValue(Object(Anexo), 'tipo_arquivo')
#3 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(534): CHtml::activeDropDownList(Object(Anexo), 'tipo_arquivo', Array, Array)
#4 C:\localhost\gertcc\protected\views\anexo\_form.php(24): CActiveForm->dropDownList(Object(Anexo), 'tipo_arquivo', Array)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#8 C:\localhost\gertcc\protected\views\anexo\create.php(15): CController->renderPartial('_form', Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('create', Array, true)
#13 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): CController->render('create', Array)
#14 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#18 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#19 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#20 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#22 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#24 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#25 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#26 C:\localhost\gertcc\index.php(13): CApplication->run()
#27 {main} REQUEST_URI=/anexo/create
2010/11/19 10:10:01 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.tipo_arquivo" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('tipo_arquivo')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(2006): CActiveRecord->__get('tipo_arquivo')
#2 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(1379): CHtml::resolveValue(Object(Anexo), 'tipo_arquivo')
#3 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(534): CHtml::activeDropDownList(Object(Anexo), 'tipo_arquivo', Array, Array)
#4 C:\localhost\gertcc\protected\views\anexo\_form.php(24): CActiveForm->dropDownList(Object(Anexo), 'tipo_arquivo', Array)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#8 C:\localhost\gertcc\protected\views\anexo\create.php(15): CController->renderPartial('_form', Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('create', Array, true)
#13 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): CController->render('create', Array)
#14 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#18 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#19 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#20 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#22 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#24 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#25 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#26 C:\localhost\gertcc\index.php(13): CApplication->run()
#27 {main} REQUEST_URI=/anexo/create
2010/11/19 10:14:32 [error] [exception.CException] exception 'CException' with message 'A propriedade "Anexo.tipo_arquivo" não está definida.' in C:\localhost\yii-1.1.4.r2429\framework\base\CComponent.php:131
Stack trace:
#0 C:\localhost\yii-1.1.4.r2429\framework\db\ar\CActiveRecord.php(110): CComponent->__get('tipo_arquivo')
#1 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(2006): CActiveRecord->__get('tipo_arquivo')
#2 C:\localhost\yii-1.1.4.r2429\framework\web\helpers\CHtml.php(1379): CHtml::resolveValue(Object(Anexo), 'tipo_arquivo')
#3 C:\localhost\yii-1.1.4.r2429\framework\web\widgets\CActiveForm.php(534): CHtml::activeDropDownList(Object(Anexo), 'tipo_arquivo', Array, Array)
#4 C:\localhost\gertcc\protected\views\anexo\_form.php(24): CActiveForm->dropDownList(Object(Anexo), 'tipo_arquivo', Array)
#5 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#6 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#7 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#8 C:\localhost\gertcc\protected\views\anexo\create.php(15): CController->renderPartial('_form', Array)
#9 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(119): require('C:\localhost\ge...')
#10 C:\localhost\yii-1.1.4.r2429\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\localhost\ge...', Array, true)
#11 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(798): CBaseController->renderFile('C:\localhost\ge...', Array, true)
#12 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(739): CController->renderPartial('create', Array, true)
#13 C:\localhost\gertcc\protected\controllers\AnexoController.php(88): CController->render('create', Array)
#14 C:\localhost\yii-1.1.4.r2429\framework\web\actions\CInlineAction.php(50): AnexoController->actionCreate()
#15 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(300): CInlineAction->run()
#16 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilter.php(41): CFilterChain->run()
#18 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(1049): CFilter->filter(Object(CFilterChain))
#19 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#20 C:\localhost\yii-1.1.4.r2429\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(283): CFilterChain->run()
#22 C:\localhost\yii-1.1.4.r2429\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(324): CController->run('create')
#24 C:\localhost\yii-1.1.4.r2429\framework\web\CWebApplication.php(121): CWebApplication->runController('anexo/create')
#25 C:\localhost\yii-1.1.4.r2429\framework\base\CApplication.php(135): CWebApplication->processRequest()
#26 C:\localhost\gertcc\index.php(13): CApplication->run()
#27 {main} REQUEST_URI=/anexo/create
